code/__DEFINES/flags.dm

```diff
@@ -168,6 +168,16 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define KNPC_IS_DOOR_BASHER 1<<4		//Does the KNPC kick the door off its hinges if it doesn't have a valid ID?
 #define KNPC_IS_DOOR_HACKER 1<<5		//Does the KNPC hack the door open if it doesn't have a valid ID?
 
+//Knpc pathfinding return values
+#define KNPC_PATHFIND_SUCCESS 0 //Path successfully generated
+#define KNPC_PATHFIND_SKIP 1 //Regenerating of path isn't needed, incapable of moving, etc
+#define KNPC_PATHFIND_TIMEOUT 2 //Pathfinding is currently in timeout due to having failed previously.
+#define KNPC_PATHFIND_FAIL 3 //No path found
+
+//Knpc pathfinding timeout defines
+#define KNPC_TIMEOUT_BASE (3 SECONDS) //The base timeout applied if an knpc's pathfinding fails.
+#define KNPC_TIMEOUT_STACK_CAP 9 //Every consecutive pathfinding fail adds a stacks; Timeout applied uses them as multiplier up to a cap of this value.
+
```

Comment:
This file is for bitflags put these regular defines somewhere else

Reply:
A seperate file in the nsv folder is also fine

Reply:
.. I thought I had those somewhere reasonable, huh..